File: lua/refactoring/tests/debug/print_var/go/multiple-statements/print_var.expected.go
package main

import "fmt"

func simple_function(a int) {
	var test int = 1
	test_other := 1
fmt.Println(fmt.Sprintf("custom print var simple_function test_other:: %v", test_other))// __AUTO_GENERATED_PRINT_VAR__
	for idx := test - 1; idx < test_other; idx++ {
		fmt.Println(idx, a)
	}
}

File: lua/refactoring/tests/debug/print_var/lua/multiple-statements/print_var.expected.lua
function simple_function(a)
    local test = 1
    local test_other = 11
print("custom print_var simple_function test_other:", vim.inspect(test_other))-- __AUTO_GENERATED_PRINT_VAR__
    for idx = test - 1, test_other do
        print(idx, a)
    end
end
